spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -218,12 +218,23 @@ def pack_measures(self):
         writer.save()
 
     def show_total_results(self):
+        """
+        Print total results to CLI
+        """
+
         printv('\n\nAveraged measures...', self.verbose, 'normal')
-        for stg, key in zip(['  Volume [mm^3] = ', '  (S-I) Length [mm] = ', '  Equivalent Diameter [mm] = '], ['volume [mm3]', 'length [mm]', 'max_equivalent_diameter [mm]']):
-            printv(stg + str(np.round(np.mean(self.measure_pd[key]), 2)) + '+/-' + str(np.round(np.std(self.measure_pd[key]), 2)), self.verbose, type='info')
 
-        printv('\nTotal volume = ' + str(np.round(np.sum(self.measure_pd['volume [mm3]']), 2)) + ' mm^3', self.verbose, 'info')
-        printv('Lesion count = ' + str(len(self.measure_pd['volume [mm3]'].values)), self.verbose, 'info')
+        for label, key in zip(['Volume [mm^3]', '(S-I) Length [mm]', 'Equivalent Diameter [mm]'], self.measure_keys):
+            mean_value = np.round(np.mean(self.measure_pd[key]), 2)
+            std_value = np.round(np.std(self.measure_pd[key]), 2)
+            measure_info = f'{label} = {mean_value} +/- {std_value}'
```

Comment:
Your refactor in 9b5ee2914e1e8731b2aa776bcabc6d0f2e8521f2 is wonderful! Things are much clearer now.

However, I notice the old `label` strings had two spaces (`'  '`) at the start of each label (e.g. `'  Volume [mm^3] = '`). So, if we want the output to be exactly the same, I think we need to do this:

```suggestion
            measure_info = f'  {label} = {mean_value} +/- {std_value}'
```